templates/index.html

```diff
@@ -122,7 +129,11 @@ <h1>MAX Image Caption Generator</h1>
 
         <footer class="footer">
             <hr/>
-            <p>Created at the IBM Center for Open-Source Data & AI Technologies</p>
+            <p style="float:left">Created at the IBM Center for Open-Source Data & AI Technologies</p>
+            <p style="float:right">
+                Source code available on
+                <a href="https://github.com/IBM/MAX-Image-Caption-Generator-Web-App" target="_blank">GitHub</a>
```

Comment:
+1! nice addition

Reply:
It's in my other web apps so I figured I should add it here too